WebFeedGatewayPlugin.php

```diff
@@ -126,7 +128,8 @@ public function fetch($args, $request): bool
                     'latestDate' => $latestDate,
                     'feedUrl' => $request->getRequestUrl(),
                     'userGroups' => $userGroups,
-                    'includeIdentifiers' => $includeIdentifiers
+                    'includeIdentifiers' => $includeIdentifiers,
+                    'language' => str_replace('_', '-', str_replace(['@cyrillic', '@latin', '_Latn', '_Cyrl'], '', Locale::getLocale())),
```

Comment:
Looks like the BCP 47 format is roughly `<language>-<script>-<region>` (there are other optional sub-components, but not important for us), so this will work for our current locales, but other variants might break this in the future, because our locale has a different order `<language>-<region>-<script>`.

Also, the `str_replace()` seems to be too limited (I guess you've extracted this from the `Locale` class, because I saw someone added it there), because there are extra scripts that we're not covering.

I think it's enough just to fix the "order".

About the limited scripts on the replace, that's something for another issue.

Reply:
Hmmm... I was only looking at our currently existing UI locales and the fact that we will move to Weblate locales soon (s. https://github.com/pkp/pkp-lib/issues/9707). 
But, maybe I should consider all Weblate locales:
I see now that Weblate has different scripts (than Latn and Cyrl) but also something with @ (e.g. hi@hinglish, pt_BR@formal, sr@ijekavian_Latn).
Regarding the order: it seems Weblate locales have the order language_script_region (e.g. en_Shaw_US, zh_Hant_HK). 
Thus, currently I am not sure what pattern would work to get it correctly. Maybe I can use PHP intl functions Locale::getPrimaryLanguage and Locale::getRegion to get it right? :thinking: